content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -14,7 +14,7 @@ the sequence of actions in a cluster. The cluster audits the activities generate
 by applications that use the Kubernetes API, and by the control plane itself.
 
 Auditing allows cluster administrators to answer the following questions:
-
```

Comment:
We'd better keep the empty line because it improves readability.
It serves a separator between the paragraph and the list.

Reply:
> We'd better keep the empty line because it improves readability. It serves a separator between the paragraph and the list.

+1